inlang/documentation/ecosystem/concept/alias.md

```diff
@@ -0,0 +1,53 @@
+# Message Alias
+
+**❗ Only use aliases if you import pre-existing messages, not for newly created messages.** 
+
+Aliases allow managing pre-existing messages with keys like "account-settings-button-label" with the inlang ecosystem by treating the key as an "alias" for a message. 
```

Comment:
how about:

> Aliases provide a way to reference messages using pre-existing message identifers. 
>
> This is useful when you import messages and your code already has references to those messages.
> If your project is configured to use a plugin for message storage, the ids from the plugin are treated as aliases.

Reply:
id != key

pre-existing messages with keys have no identifier. that's the problem. i don't think we should mix id and key interchangeably